### ISSUE TITLE
[EEP 153: Norman Borlaug] Code Review for TW Schultz

### ISSUE THREAD

**Issue by vishalival** [neha-lala/TWSchultz#1]
Great topic, and love the population breakdown! I have some comments on the code file `EEP153 P4.1 (1) (1).ipynb`

I love this graph, very nice idea and love the colors. It will look amazing in your final presentation!
https://github.com/neha-lala/TWSchultz/blob/132c0e85106ef54e35bbc482f1168608ceac6d16/EEP153%20P4.1%20(1)%20(1).ipynb#L3705

I can't really see any assert statements in the Unit Tests section, I only see the data being read in. Everything looks great, I see that you are still working on the policy idea - so some suggestions would be a tax/subsidy, food fortification (more fiber into bread, magnesium, potassium), among others. Another suggestion is to add titles to some of the graphs (the histograms) and also adding a median line, like for this graph:

https://github.com/neha-lala/TWSchultz/blob/132c0e85106ef54e35bbc482f1168608ceac6d16/EEP153%20P4.1%20(1)%20(1).ipynb#L2821

So, you can plot a median line (so we can clearly see median nutritional outcome for every nutrient) with this line of code:
`result = plt.hist(x, bins=20, color='c', edgecolor='k', alpha=0.65)

plt.axvline(x.mean(), color='k', linestyle='dashed', linewidth=1)`
So basically add the axv line

The Engel curve here is hard to read with the super long legend, so maybe you can filter it to be the most consumed goods?

https://github.com/neha-lala/TWSchultz/blob/132c0e85106ef54e35bbc482f1168608ceac6d16/EEP153%20P4.1%20(1)%20(1).ipynb#L3584
